src/index.js

```diff
@@ -3,8 +3,11 @@
 
 // An example of how you tell webpack to use a CSS (SCSS) file
 import './css/base.scss';
+import './recipe.js';
```

Comment:
We'll review here to get our api calls going, in the next few days, good call to keep the data coming in, for now, from elsewhere.